src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -125,19 +128,36 @@ private function getEnvFiles(): array
         return $files;
     }
 
-    private function getFilePath(string $file): string
+    private function getFilePath(?string $dotenvDirectory, string $file): string
```

Comment:
The signature of this method don't need to be modified once `$file` contains the directory.